cle/backends/elf.py

```diff
@@ -211,6 +212,13 @@ def __init__(self, binary, **kwargs):
 
         self.__parsed_reloc_tables = set()
 
+        # The linked image base should be evaluated before registering any segment or section due to
+        # the fact that elffile, used by those methods, is working only with un-based virtual addresses, but Clemories
+        # themselves are organized as a tree where each node backer internally uses relative addressing
+        self.mapped_base = self.linked_base = min(
+            map(lambda x: x['p_vaddr'],
+                filter(lambda x: x.header.p_type == 'PT_LOAD', self.reader.iter_segments())) or [0]
+        )
```

Comment:
From specification:
> To compute the base address, one determines the memory address associated with the lowest p_vaddr value for a PT_LOAD segment. _One then obtains the base address by truncating the memory address to the nearest multiple of the maximum page size._ 
Depending on the kind of file being loaded into memory, the memory address might or might not match the p_vaddr values.

So resulting minimum should be only additionally aligned on a page size, which is actually the same what you're doing at `ELF._load_segment_memory`:
```python
    def _load_segment_memory(self, seg):

        # see https://code.woboq.org/userspace/glibc/elf/dl-load.c.html#1066
        ...
```